README.md

```diff
@@ -34,8 +34,14 @@ It came from a desire to quickly dive into graphics and performance experiments.
 * Easy-to-use dynamic constant buffers and descriptor tables
 
 ## Requirements
-* Windows 10 with the May 2019 Update
-* [Visual Studio 2019](https://www.visualstudio.com/) with the [Windows 10 May 2019 Update SDK (18362)](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
+### Master branch
+This branch is intended for the latest [released](https://docs.microsoft.com/en-us/windows/release-information/) Windows 10 version.
+* Windows 10 version 1909
+* [Visual Studio 2019](https://www.visualstudio.com/) with the [Windows 10 SDK version 1903(18362)](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
```

Comment:
The SDK reference here is wrong... It's Windows 10 SDK (19041)

Reply:
I'll keep the master branch as-is for now since the 19041 SDK isn't released yet. Latest released SDK (outside of Windows Insiders Program) is still 18362.